### PR TITLE
challenge submission

### PR DESCRIPTION
**Pull request by jameslaw195**

### CODE REVIEW

File: css-basics/css-basics_button-styles/css/styles.css [hunk 1]
```diff
@@ -30,19 +30,23 @@ button {
 /* Add your CSS styles below this line. Feel free to replace the comments with your code. Make sure not to put your code into the comments, because then nothing will happen. */
 
 .button1 {
-  /*Style me! Change my border, set a border radius or what you want to try out. :) */
+  border: 6px solid blue;
```
Comment (CheckersRT) [inline css-basics/css-basics_button-styles/css/styles.css:33]: The instructions mention a border radius as well, if you feel so inclined!

File: css-basics/css-basics_button-styles/css/styles.css [hunk 2]
```diff
@@ -30,19 +30,23 @@ button {
...
 
 .button3 {
-  /*Style me! You can test out changing the font, or font size, ...*/
+  font-family: "Courier New", Courier, monospace;
+  font-size: x-large;
 }
 
 .button4 {
   /*Style me! Can you add a shadow to this button? Maybe play around with the shadow properties, as well? You can even color the shadow, make it bigger or smaller, ... */
+  box-sizing: border-box;
```
Comment (CheckersRT) [inline css-basics/css-basics_button-styles/css/styles.css:48]: Did this do anything? I think you would only notice the effect if you changed the sizing as well.

**Review by CheckersRT — COMMENTED** [review]
Spiffing m'dear. Couple of comments, if you'd indulge me.

File: css-basics/css-basics_button-styles/css/styles.css (hunk 2 above)
Comment (jameslaw195) [reply css-basics/css-basics_button-styles/css/styles.css:48]: Hey yes this was enough to add a shadow!

**Pull request merged**